scripts/globals/mannequins.lua

```diff
@@ -0,0 +1,59 @@
+----------------------------
+-- Information on Mannequins
+----------------------------
+
+require("scripts/globals/items")
+
+xi = xi or {}
+xi.mannequin = xi.mannequin or {}
+
+xi.mannequin.type =
+{
+    HUME_M   = 0x01,
+    HUME_F   = 0x02,
+    ELVAAN_M = 0x04,
+    ELVAAN_F = 0x08,
+    TARU_M   = 0x10,
+    TARU_F   = 0x20,
+    MITHRA   = 0x40,
+    GALKA    = 0x80,
+}
+
+xi.mannequin.pose =
+{
+    NORMAL    = 0x00,
+    SIT       = 0x01,
+    SALUTE_S  = 0x02,
+    SALUTE_B  = 0x03,
+    SALUTE_W  = 0x04,
+    HURRAY    = 0x08,
+    SPECIAL   = 0x10,
+}
+
+xi.mannequin.cost =
+{
+    PURCHASE  = 100000,
+    TRADE     = 2000,
+    POSE      = 200,
+}
+
+xi.mannequin.getMannequins = function(player)
+    local man = 0
+    for i = 0, 7 do
+        -- Cycle through all 8 mannequin options and see if the player has them
+        if player:hasItem(xi.items.HUME_M_MANNEQUIN + i) then
+            man = man + (2 ^ i)
```

Comment:
Instead of adding here, I would suggest using `utils.mask.setBit` function to make it obvious that its a bitfield.  Also, instead of `i` could you rename the iterator to something more obvious? (itemOffset, etc).  Personally, I'd attack it by iterating over the items, and then subtracting the item to get the bit (ex below)

```
for itemId = xi.items.HUME_M_MANNEQUIN, xi.items.GALKA_MANNEQUIN do
    if player:hasItem(itemId) then
        mannequinMask = utils.mask.setBit(mannequinMask, itemId - xi.items.HUME_M_MANNEQUIN, true)
    end
end
```